src/index.js

```diff
@@ -12,7 +12,6 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 import './index.css';
-import 'react-confirm-alert/src/react-confirm-alert.css';
```

Comment:
Isn't it useful ?

Reply:
useless, replace by a component from materail-ui